### PR TITLE
Adds new metrics for tracking deactivation of old plugin

### PR DESCRIPTION
**Pull request by aslo**
fixes https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/892

### CODE REVIEW

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java [hunk 1]
```diff
@@ -109,6 +115,9 @@ public void hyperlinkUpdate(@NotNull Notification notification, @NotNull Hyperli
       String href = event.getDescription();
 
       if (DEACTIVATE_LINK_HREF.equals(href)) {
+        UsageTrackerProvider.getInstance()
+            .trackEvent(GctTracking.APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK)
```
Comment (etanshaul) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java]: whats the difference between APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK and APP_ENGINE_OLD_PLUGIN_DEACTIVATED ?

Comment (etanshaul) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java]: Oh is the notification_click clicking on the warning to open up the "are you sure you want to disable.." dialog?

Comment (aslo) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java]: `APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK` tracks when a user clicks the link in the notification:



Clicking that link opens a dialog which prompts the user to uninstall the plugin (and optionally restart the application). That click is tracked by `APP_ENGINE_OLD_PLUGIN_DEACTIVATED`.

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java [hunk 2]
```diff
@@ -47,4 +47,11 @@
       "cloud.debugger.close.continue.listen";
   public static final String CLOUD_DEBUGGER_NOTIFY_BREAKPOINT_LIST_CHANGE =
       "cloud.debugger.notify.breakpoint.list.change";
+
+  public static final String APP_ENGINE_OLD_PLUGIN_NOTIFICATION =
+      "appengine.oldplugin.notification";
+  public static final String APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK =
```
Comment (patflynn) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java]: what do you think about renaming this to appengine.oldplugin.disable.click?  presumably they can choose not to disable?

Comment (etanshaul) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java]: appengine.oldplugin.disable.click makes me think its been disabled (and this is just to click to confirm disabling).

Comment (aslo) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java]: Well, that's actually tracked by `appengine.oldplugin.deactivated` below. This event is used to track a click on the link in the notification that's shown to the user (which just opens a dialog with more detail, and the button that actually disables the plugin). See my above comment for more detail.

**Comment by patflynn** [comment]
I would really love to get this in before our Beta release. Any chance we can merge this and cut a new release build?

**Comment by etanshaul** [comment]
Yes. I'd like to also include: https://github.com/GoogleCloudPlatform/google-cloud-intellij/pull/899

**Comment by etanshaul** [comment]
I'm ok with the current state of this. lgtm.

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java (hunk 2 above)
Comment (patflynn) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/GctTracking.java]: @aslo I think that the name of this label is too broad. It sounds like you're clicking on the notification, but we want to specifically know that the 'disable plugin' link was clicked.

**Comment by patflynn** [comment]
I'm good with this once the disable link click event label has been made a bit more specific.

**Comment by aslo** [comment]
Updated to address @patflynn's concern. Feel free to merge once tests pass, in case I'm AFK when it's done.

**Pull request merged**